protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -554,6 +577,26 @@ func renderMessagesAsDefinition(messages messageMap, d openapiDefinitionsObject,
 	return nil
 }
 
+func checkVisibility(r *visibility.VisibilityRule, reg *descriptor.Registry) bool {
```

Comment:
Please add a comment to this method. The name is a bit confusing to me, I'd prefer it to be something like `isVisible` since it returns a bool, what do you think?